### PR TITLE
MP RIO Fix

### PR DESCRIPTION
**Pull request by SammySkycrafts**
MP Rio fixed
Added missing properties due to earlier damage.nas upgrade

### CODE REVIEW

File: aircraft/f-14b/f-14b-bs-set.xml [hunk 1]
```diff
@@ -890,6 +890,9 @@
             <station-name type="string">station</station-name>
             <hit-interpolation type="bool">true</hit-interpolation>
             <enable-craters type="bool">true</enable-craters>
+            <!--MP-lat type="string">rotors/main/blade[0]/flap-deg</MP-lat>
+            <MP-lon type="string">rotors/main/blade[1]/flap-deg</MP-lon>
+            <MP-alt type="string">rotors/main/blade[2]/flap-deg</MP-alt-->
```
Comment (Zaretto) [inline aircraft/f-14b/f-14b-bs-set.xml]: Is there a reason to include commented out code here?

**Comment by Zaretto** [comment]
Is this to fix tacview

**Comment by SammySkycrafts** [comment]
> Is this to fix tacview

Makes the MP RIO's radar work again, which was due to the `tacview.nas` file being missing. The radar code calls that file, so it is either add in the tacview files to the bs-set, or maintain an additional radar nasal file that is the same as the A/B, but without the tacview=1 which could lead to it breaking further if one is updated but nit the other.

**Comment by Zaretto** [comment]
Thanks for the fix

**Pull request merged**